### DISCUSSION TITLE
Disable standby mode on mobile/tablet

### DISCUSSION THREAD

**Discussion by floreo** [mealie-recipes/mealie#937] [Feature Request]
**Before submitting this feature request I have**

- [X] Reviewed the documentation and verified that my feature request isn't already a feature.
- [X] Reviewed the existing feature requests and my feature has not been requested.
- [X] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

Hello,

when cooking, it would be useful to disable standby mode on mobile/tablet ("Désactiver la veille" on my screenshot) so you can cook without having to unlock.
Website [jow](https://jow.fr/recipes/pasta-poulet-citron-82mtjfm0hw5s00y307hb?covers=1) has this feature which looks like this:
![Screenshot_20220116-105627](https://user-images.githubusercontent.com/8515281/149656415-1955e850-3dd5-4eda-b0e5-7fbaa9e08b61.png)
Could it be possible to add this feature ?
Thanks.

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by Sennevds** [comment]
That's interesting. Looks like they are using: https://github.com/richtr/NoSleep.js

**Comment by zierbeek** [comment]
I like this idea a lot!

**Comment by Symbiot78** [comment]
is this anything that has been considered? many many webpages with recipes has this option to keep screen active

**Comment by Sennevds** [comment]
It's already included but you need a ssl certificate to let it work

**Reply by Symbiot78** [reply]
hmm... is there something in the docs for this?
I connect via letsencrypt/cloudflare using https ..

**Reply by hay-kot** [reply]
We're using a web API to keep the screen awake, if your device does not support the API it won't work.

**Comment by hay-kot** [comment]
This feature is included in Mealie v1 provided your device supports the API

https://developer.mozilla.org/en-US/docs/Web/API/Screen_Wake_Lock_API

**Comment by Symbiot78** [comment]
@hay-kot  -  can you explain how it works? is it just there.. no user interaction needed? or ? 
I'm accessing mealie via chrome on ios17 through letsencrypt/cloudflare etc - so I would think that the cert part should be ok - but obviously I am not seeing the effect on my device :)

**Reply by boc-the-git** [reply]
There's a known issue re iOS specifically. You should be able to find it.

**Comment by Symbiot78** [comment]
@boc-the-git  - thanks. I found the issue (and the 'stay alive' button. I had never thought to look in the very bottom of the page.

**Comment by Symbiot78** [comment]
AND @hay-kot  / @boc-the-git  - it works in chrome & safari both. 
ios 17.4

I will add a very meek and "this is definetely not a complaint" feature request to move the 'keep awake' button to the top - in my honest opinion it is malplaced in the very bottom of the recipe page.

**Reply by boc-the-git** [reply]
My counter comment to that would be does it not retain the setting? I feel like it's just always on for me, I never have to toggle it. So using premium real estate up top would be a waste.

(not saying there isn't room to improve it, just needs proper analysis)
